draft/security.md

```diff
@@ -0,0 +1,109 @@
+- Provide security advice to Django package maintainers when requested
+- Assist with security aspects of the djangoproject.com website
+- Mentor new contributors interested in Django security
+- Run or support security-focused sessions in DjangoCon sprints
+- Chair, Co-Chair and Board Liaison can sign off on security decisions
+
+## Initial membership
+
+- Chair: TBD
+- Co-Chair: TBD
+- Board Liaison (must be an active Board member; may be the same as Chair/Co-Chair): TBD
+- Other members:
+  - TBD
+  - ...
+
+## Future membership
+
+### Who is eligible to join? Any volunteer, or are there specific requirements?
+
+Members must have interest in Django security and should have experience with web security concepts, particularly as they relate to Django applications. Prior experience with Django and its security mechanisms (e.g., CSRF protection, XSS protection) is highly recommended. Members should be familiar with the Django [security release process](https://docs.djangoproject.com/en/dev/internals/security/) or willing to learn.
+
+We welcome contributors with various levels of experience, but due to the sensitive nature of security work, members should demonstrate a track record of responsible security practices. Experience with security vulnerability assessment, analysis, or remediation is a plus.
+
+### Access granting and deployment
+
+Initial onboarding phase where only Chair and Co-Chair have permissions to access security reports and coordinate responses. After onboarding of 6 months, other group members who have proven their expertise will be granted access to the security issue tracker and response coordination at the discretion of the chair and co-chair.
+
+The deployment of security fixes will be coordinated with the Django release team and the ops team.
+
+### How do people who want to join sign up / volunteer / express interest?
+
+Individuals can express interest by opening a PR to this working group repository using a template, adding their names in the list of members along with a brief description of their security experience and motivation to join the group.
+
+Alternatively, individuals can send an email to the Chair or Co-Chair expressing interest and outlining their qualifications and motivation.
+
+### How will decisions on adding/removing members be handled?
+
+Direct membership: new members may self-nominate; the WG will vote (50%+1) to approve/deny new members. The WG will vote for New Chair/Co-Chairs and decision to appoint will be based on gaining majority votes.
+
+Members join the group for one year term. At the end of this term, they need to opt into staying involved to keep being a member of the group.
+
+Due to the sensitive nature of security work, members who fail to adhere to responsible disclosure practices or exhibit other concerning behavior may be removed by a vote of the working group (50%+1) or by decision of the Chair and Co-Chair in consultation with the Board Liaison.
```

Comment:
Issue: I think for the sake of Django's security, this needs to be a little stricter.
```suggestion
Due to the sensitive nature of security work, members who fail to adhere to responsible disclosure practices or exhibit other concerning behavior will be removed by a vote of the working group (50%+1) or by decision of the Chair and Co-Chair in consultation with the Board Liaison.
```